python/python/bystro/ancestry/train.py

```diff
@@ -166,12 +166,20 @@ def _parse_vcf_line_for_dosages(
     if variant in variants_to_keep:
         fields = line.split()  # now we can parse the full line
         variant_dosages = [
-            int(psa[0]) + int(psa[2]) for psa in fields[NUM_VCF_METADATA_COLUMNS:]
-        ]  #  pipe-separated annotation e.g. '0|1'
+            _parse_genotype_field(field) for field in fields[NUM_VCF_METADATA_COLUMNS:]
+        ]  #  genotype fields take the form e.g. '0|1', '0/1' or './.'
```

Comment:
I think one dot like this '.' is also an option for missing genotypes, do we want to be able to handle that format?

Reply:
Yes, we do handle that format and test for it in the tests.  But will update the docstring to make that clearer, ty.